feature/system/gnmi/cliorigin/ate_tests/cli_origin_test/cli_origin_test.go

```diff
@@ -129,6 +131,16 @@ func TestOriginCliConfig(t *testing.T) {
 	}
 }
 
+func nokiaCLI(intf string, enabled bool) string {
+	op := "disable"
+	if enabled {
+		op = "enable"
+	}
+	return fmt.Sprintf(`
+   interface %s admin-state %s
```

Comment:
shall we add `/` before the interface just to be precise about the context?

Reply:
You're right, it makes context more precise - I'll add.